relay-general/src/pii/regexes.rs

```diff
@@ -269,6 +269,6 @@ static US_SSN_REGEX: Lazy<Regex> = Lazy::new(|| {
 
 static PASSWORD_KEY_REGEX: Lazy<Regex> = Lazy::new(|| {
     Regex::new(
-        r"(?i)(password|secret|passwd|api_key|apikey|auth|credentials|mysql_pwd|privatekey|private_key|.*token.*)"
+        r"(?i)(password|secret|passwd|api_key|apikey|auth|credentials|mysql_pwd|privatekey|private_key|set-cookie|.*token.*)"
```

Comment:
Something I missed on the previous PR: If `auth` matches `Proxy-Authorization`, then `token` matches `whatever_token_123` ... so I think we can omit the `.*`
```suggestion
        r"(?i)(password|secret|passwd|api_key|apikey|auth|credentials|mysql_pwd|privatekey|private_key|set-cookie|token)"
```

Reply:
good point! done in [8e37424](https://github.com/getsentry/relay/pull/1529/commits/8e37424ff871ea3e8dfc5ffd50c6b8dd86ce7a63)